test-dataset-plan.md

```diff
@@ -0,0 +1,39 @@
+# Plan for the test dataset
+
+We want the test dataset to contain an equal amount of data split into various
+tasks, described below.
+
+Aside from ensuring that the resulting models perform well across multiple
+tasks and do not overfit to any single one such, this would also provide
+transparent performance descriptions of the trained models. Such transparency
+would mean that anyone who is looking for a language model to finetune on their
+dataset can immediately see whether the model works well on their domain/task.
+
+## Tasks
+
+As mentioned above, we want the test dataset to contain an equal amount of
+data from each of the following tasks:
+
+- Danish web data (Netarkivet)
+- Danish social media data (HOPE Twitter, Reddit-da and Hestenettet from Danish
+  Gigaword)
+- Danish legal data (Retsinformation and Skat.dk, both from Danish Gigaword)
+- Danish wiki data (Danish Wikipedia and lex.dk)
+- Danish news data (DaNews)
+- Danish books (Gutenberg, Danish literature, WikiBooks and WikiSource, all
+  from Danish Gigaword)
+- Danish spontaneous text (OpenSubtitles and Spontaneous Speech, both from
```

Comment:
Is relevant to include FT speech because it is typically used for speech to text?

Reply:
FT = folketinget

Reply:
Speeches could potentially make sense, but I don't want it to be included in the spontaneous text, since it's the opposite. The reason why spontaneous text could be interesting is because of all the discontinuities ("Hej, øh, jeg hedder Dan, hvad hed-, jeg mener, øh, hvem er du?")

Reply:
Speeches could be interesting - agree on placing them in another group than spontaneous. FT Speech has some weird pecularities so it might be better to use speeches from e.g. the prime minister (I scraped them at some point so have them lying around).

Reply:
So keep spont. speech and speeches?

Reply:
@MalteHB There is 700kb spontaneous speech in DAGW, and you get a bunch from web data, Twitter data, reddit and hestenettet, too. I'm not worried about a lack of training data, just that we choose the right test data :)